poofarmer/project.godot

```diff
@@ -29,7 +29,7 @@ _global_script_classes=[ {
 "language": "GDScript",
 "path": "res://Scripts/Gnome.gd"
 }, {
-"base": "Enemy",
+"base": "KinematicBody2D",
```

Comment:
Was this intentional?

Reply:
No I think this might have been old stuff in my project.godot

Reply:
We _want _ this to be an `Enemy` right?

Reply:
Yes

Reply:
I see now, I at first thought this was setting types up in the Main scene, but I see that it was setting the types for the scenes in the whole project